sympy/crypto/crypto.py

```diff
@@ -28,7 +28,7 @@
 from sympy.polys.polytools import gcd, Poly
 from sympy.utilities.misc import filldedent, translate
 from sympy.utilities.iterables import uniq
-from sympy.utilities.randtest import _randrange
+from sympy.utilities.randtest import _randrange, _randint
```

Comment:
These functions are supposed to be used for the test suite. This is the same as just using `random.Random`.

Reply:
So should I directly use the random.Random? I saw that in GM cryptosystem _randrange was used.